src/Security/Authentication/Cookies/src/CookieAuthenticationEvents.cs

```diff
@@ -109,10 +109,16 @@ private static bool IsAjaxRequest(HttpRequest request)
     }
 
     /// <summary>
-    /// Invoked to validate the prinicipal.
+    /// Invoked to validate the principal.
     /// </summary>
     /// <param name="context">The <see cref="CookieValidatePrincipalContext"/>.</param>
     public virtual Task ValidatePrincipal(CookieValidatePrincipalContext context) => OnValidatePrincipal(context);
+    
+    /// <summary>
+    /// Invoked to check if the cookie should be renewed.
+    /// </summary>
+    /// <param name="context">The <see cref="CookieSlidingExpirationContext"/>.</param>
+    public virtual Task CheckSlidingExpiration(CookieSlidingExpirationContext context) => OnCheckSlidingExpiration(context);
```

Comment:
This isn't being called anywhere, update the call site:
https://github.com/dotnet/aspnetcore/blob/c85baf8db0c72ae8e68643029d514b2e737c9fae/src/Security/Authentication/Cookies/src/CookieAuthenticationHandler.cs#L96